app/views/katello/api/v2/content_facet/show.json.rabl

```diff
@@ -25,6 +25,14 @@ child :content_facet => :content_facet_attributes do
   node :katello_tracer_installed do |content_facet|
     content_facet.tracer_installed?
   end
+
+  node :katello_agent_enabled do
```

Comment:
I know as a dev it's easy to add these here our sake but it feels a bit weird. Random idea: could the /katello/api/v2 (or similar endpoint) expose some amount of configuration? Multiple places may need this info and a content facet may not be in scope at that point (I don't have an example off the top of my head)?

Or maybe Foreman or other plugin has established something we can use?

Reply:
This was discussed also. I could separately get the configuration separately but having that information embedded in the host details made best sense for me. Given we already have `katello_tracer_installed` and `katello_agent_installed` there. I dont see why adding `katello_agent_enabled` and `remote_execution_by_default` attributes should be a red flag. Considering these are details pertinent to this host?

Reply:
The examples you gave are reflective of the state of the content host - whether tracer or agent is installed. The fields you've added are related to the configuration of Katello which I guess is pertinent but also has nothing to do with the content facet itself. If you and Jeremy are good then let's go with it

Reply:
I think ForemanContext would be the ideal place to have something like this. It already contains other global app settings. The problem is that adding values there requires Foreman code changes, and also we'd be adding Katello-specific data.  Until we can come up with a way to address both of those issues, I think adding it here is... okay but not great.  Adding to the / API endpoint is an interesting idea too, but then we'd have to make an additional API call.

I think going with what we have is fine for now, because at least it's an API call that we're already making.  Unless someone can think of another brilliant idea :)

Reply:
First time I've seen ForemanContext and it really does seem like the right place. I'm surprised that component wasn't designed with plugin extensibility in mind. That is something we should look into in the future!